src/Features/Core/Portable/Diagnostics/Analyzers/NamingStyles/Serialization/SymbolSpecification.cs

```diff
@@ -202,31 +201,20 @@ public bool AppliesTo(ISymbol symbol)
             }
 
             internal XElement CreateXElement()
-            {
-                if (SymbolKind.HasValue)
-                {
-                    return new XElement(nameof(SymbolKind), SymbolKind);
-                }
-                else
-                {
-                    return new XElement(nameof(TypeKind), TypeKind);
-                }
-            }
+                => SymbolKind.HasValue
+                     ? new XElement(nameof(SymbolKind), SymbolKind)
+                     : new XElement(nameof(TypeKind), TypeKind);
 
             internal static SymbolKindOrTypeKind AddSymbolKindFromXElement(XElement symbolKindElement)
-            {
-                return new SymbolKindOrTypeKind((SymbolKind)Enum.Parse(typeof(SymbolKind), symbolKindElement.Value));
-            }
+                => new SymbolKindOrTypeKind((SymbolKind)Enum.Parse(typeof(SymbolKind), symbolKindElement.Value));
```

Comment:
Not directly related to your change, but these `Enum.Parse` calls from arbitrary XML that might have been edited by the user (in the case of profiles) worry me.